docs/fleet/fragments.md

```diff
@@ -300,6 +300,9 @@ Here are the version options:
 {{% /tab %}}
 {{< /tabs >}}
 
+After configuring overwrites, check your machine's [**LOGS** tab](/cloud/machines/#logs).
+If there are problems with overwrites to the fragment, the overwrites will not be partially applied and the configuration changes will not take effect until the configuration is fixed.
```

Comment:
"will not be partially applied" is a confusing phrase. Will they be fully applied? Not applied at all?

Reply:
You could say instead "the overwrite will fail and your config will not be returned to your machine; your robot will instead rely on its cached config until you fix the overwrite error and the next config fetch is successful".